aries-site/src/pages/components/button.js

```diff
@@ -30,6 +31,14 @@ const Button = () => (
           <ButtonExample />
         </Example>
       </Subsection>
+      {relatedContent.length > 0 ? (
+        <Subsection name="Related">
```

Comment:
Can we make this a `RelatedContentSection` component if the text will be consistent across pages?

Reply:
The H2 will likely be consistent, but I think we will need flexibility in the subsection text. I originally headed that direction, but couldn't make text that works for a component like button, also work for something more abstract like principles or topics in the Extend section.